README.md

```diff
@@ -83,95 +194,103 @@ and build any dependency needed before running your command.
 nx run dialtone-css:build
 ```
 
-#### Don't
+##### Don't
 
 Try installing packages with NX, this doesn't work at all, please use PNPM instead.
 
 ```bash
 nx add eslint --filter dialtone-icons
 ```
 
-## Quick start
+#### Running the projects
 
-If you would like to contribute to Dialtone without having to do any local environment setup, you can use GitHub
-Codespaces. You can initialize a new Codespace by clicking the green "Code" button at the top right of the Dialtone
-GitHub page.
-
-![Creating a codespace](./.github/new_codespace.png)
-
-Please see the [Codespaces docs](./.github/codespaces.md) for more information.
+First, install the dependencies for all the monorepo packages and apps.
 
-### Local environment setup
+```bash
+pnpm install
+```
 
-- We use [pnpm](https://pnpm.io) for managing dependencies, so you need to have it installed
-in order to be able to manage the packages. In order to install it, run the following command
-or follow its [installation guide](https://pnpm.io/installation):
+##### Dialtone documentation site
 
 ```bash
-npm install -g pnpm
+nx start:dialtone
 ```
 
-Once pnpm is installed, install nx globally to make sure you can run commands such as `nx run build`
-without having to prefix them with `pnpm` or `pnpm exec`:
+This will start the documentation site and watch the library for changes, it will be live updated with any changes.
+
+Access the local server at `http://localhost:4000`
+
+##### Dialtone Vue 2 storybook
 
 ```bash
-pnpm add --global nx@latest
+nx start:dialtone-vue2
 ```
 
-Then, install the dependencies for all the monorepo packages and apps.
+Access the local storybook server for Dialtone Vue 2 via `http://localhost:9010/`
+
+##### Dialtone Vue 3 storybook
 
 ```bash
-pnpm install
+nx start:dialtone-vue3
 ```
 
-### Running the projects
+Access the local storybook server for Dialtone Vue 3 via `http://localhost:9011/`
+
+#### Common Commands
 
-#### Dialtone
+##### Production build the root project
 
 ```bash
-nx start:dialtone
+nx build
 ```
 
-This will start the documentation site and watch the library changes, so it is live updated.
+##### Run all Vue unit tests
 
-Access the local server at `http://localhost:4000`
+```bash
+nx test:vue
```

Comment:
Just running `nx test` will run the tests as it is listed on `./project.json` and also that's the right way to do it because we can define if something needs to be build before running tests. running `nx test:vue` will run the tests directly without looking into dependencies and that might cause issues.

```suggestion
nx test
```

Reply:
oh sweet didn't know it worked like that. Yeah thats definitely the way to go.